### ISSUE TITLE
Delete

### ISSUE THREAD

**Issue by Rohan-cod** [Rohan-cod/Music_Playlist#6]
Add functionality to delete playlists.

**Comment by Vrindagupta6828** [comment]
@Rohan-cod I would like to work on this

**Comment by Rohan-cod** [comment]
Go ahead @Vrindagupta6828 
👍

**Comment by Vrindagupta6828** [comment]
@rohan-cod can you please guide me how to do it?

**Comment by Rohan-cod** [comment]
Apple music api currently only allows you to add songs to the playlist. They haven't added the functionality for PUT, DELETE, etc yet. 

But the Spotify web api has this functionality. So you can implement the feature for it. 

Refer [this](https://developer.spotify.com/documentation/web-api/reference/playlists/remove-tracks-playlist/) for more information.

**Comment by Vrindagupta6828** [comment]
Hey i am unable to solve the issue or find enough resources to learn.. @Rohan-cod can you please guide me

**Comment by Rohan-cod** [comment]
Hi @Vrindagupta6828, have a look at "create_playlist" and "get_playlist" in "[spotify.py](https://github.com/Rohan-cod/Music_Playlist/blob/master/songs/spotify.py)". The code to delete a playlist will be similar.

**Comment by Vrindagupta6828** [comment]
@Rohan-cod We have the same code in apple_music.py as well

**Comment by Rohan-cod** [comment]
Yes, but Apple Music API doesn't have the functionality to delete a playlist as I told you before. You have to implement it for Spotify only.

**Comment by Vrindagupta6828** [comment]
yes but how that's what i am asking you you said the code is similar to create playlist and get playlist. And do i have to implement it for spotify or apple_music?

**Comment by Rohan-cod** [comment]
Code will be similar because I have used the "requests" library. The only difference will be in the URL and the parameters. And once again, Apple Music API doesn't have the functionality to delete a playlist so there is no way of implementing the functionality for it. You have to implement the functionality for Spotify only. I hope it's clear now.
